packages/flutter/lib/src/animation/scroll_behavior.dart

```diff
@@ -105,26 +105,25 @@ class UnboundedBehavior extends ExtentScrollBehavior {
 Simulation _createFlingScrollSimulation(double position, double velocity, double minScrollOffset, double maxScrollOffset) {
   double startVelocity = velocity * _kSecondsPerMillisecond;
 
-  // Assume that we're rendering at atleast 15 FPS. Stop when we're
-  // scrolling less than one logical pixel per frame. We're essentially
-  // normalizing by the devicePixelRatio so that the threshold has the
-  // same effect independent of the device's pixel density.
-  double endVelocity = 15.0 * ui.window.devicePixelRatio;
+  // Assume that we're rendering at >= 15 FPS. Stop when we're
+  // scrolling less than one physical pixel per frame, which isn't
+  // visually detectable.
+  double endVelocity = 15.0 / ui.window.devicePixelRatio;
```

Comment:
I would rephrase this. It's not that we're assuming that we're getting at least 15fps. It's "Assume that if we move less than one device pixel in 66ms, the user won't notice if we just snap the animation to the end and call it a day".

...and I think 66ms is probably too short. I think we should stop only when we reach about 1 device pixel per 500ms.

I think I would set this to `1 / (0.500 * ui.window.devicePixelRatio) // logical pixels per second`

Anyway. I don't feel strongly about the value, but either way please do per the units in a comment.